Userland/Applications/TerminalSettings/TerminalSettingsView.gml

```diff
@@ -0,0 +1,91 @@
+@GUI::Widget {
+    fill_with_background_color: true
+
+    layout: @GUI::VerticalBoxLayout {
+        margins: [10]
+        spacing: 5
+    }
+
+    @GUI::GroupBox {
+        title: "Background Opacity"
+        fixed_height: 100
+
+        layout: @GUI::VerticalBoxLayout {
+            margins: [16, 8, 8]
+            spacing: 16
+        }
+
+        @GUI::Label {
+            text: "Smaller values makes the terminal appear more transparent."
+            text_alignment: "TopLeft"
+        }
+
+        @GUI::OpacitySlider {
+            name: "background_opacity_slider"
+            min: 0
+            max: 255
+            orientation: "Horizontal"
+        }
+    }
+
+    @GUI::GroupBox {
+        title: "Terminal Font"
+        fixed_height: 100
+
+        layout: @GUI::VerticalBoxLayout {
+            margins: [16, 8, 8]
+            spacing: 16
+        }
+
+        @GUI::Label {
+            text: "By default, this is the system-wide fixed-width font."
+            text_alignment: "TopLeft"
+        }
+
+        @GUI::Widget {
+            shrink_to_fit: true
+
+            layout: @GUI::HorizontalBoxLayout {
+                spacing: 6
+            }
+
+            @GUI::Frame {
+                background_role: "Base"
+                fill_with_background_color: true
+
+                layout: @GUI::VerticalBoxLayout {
+                }
+
+                @GUI::Label {
+                    name: "terminal_font_label"
+                    text: "Csilla 10 400"
+                }
+            }
+
+            @GUI::Button {
+                text: "..."
+                name: "terminal_font_button"
+                fixed_width: 30
+            }
+        }
+    }
+
+    @GUI::GroupBox {
+        title: "Color Scheme"
+        fixed_height: 100
+
+        layout: @GUI::VerticalBoxLayout {
+            margins: [16, 8, 8]
+            spacing: 16
+        }
+
+        @GUI::Label {
+            text: "Depending on your theme, color schemes may not be available."
```

Comment:
If color schemes are not available due to `$reason`, we can show a label explaining why.
If they are available, we don't need to show anything.